static/website/templates/get-question.html

```diff
@@ -91,8 +97,14 @@
 
         <span class="modify">
             {% if user|can_edit:question and not ans_count or user|is_moderator and MODERATOR_ACTIVATED %}
```

Comment:
use brackets here also (though not part of this change)

Reply:
Cann't use brackets in if condition in templates

Reply:
```suggestion
            {% if can_edit_question(user, ans_count) %}
```

in that case, please define a function with appropriate parameters and call that function from here. Suggested code (please fix and use):

```
def can_edit_question(user, ans_count):
    if user|can_edit:question and not ans_count:
         return True
   if user|is_moderator and MODERATOR_ACTIVATED:
        return True
   return False
```